server/api/organisation/organisation.validation.js

```diff
@@ -0,0 +1,24 @@
+const websiteRegex = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w.-]+)+[\w\-._~:/?#[\]@!$&'()*+,;=.]+$/
```

Comment:
As the `regex` for validating an organisation's website is now here, should `OrgDetailForm` be updated to using this regex to avoid duplication?

Reply:
I removed the website regex in the organisation validation.js